### PR TITLE
nmea0183: Add parsing of Quectel PQTMGPS message

### PR DESCRIPTION
**Pull request by Qbicz**

### CODE REVIEW

**Comment by Qbicz** [comment]
Ready to review

File: drivers/driver_nmea0183.c [hunk 1]
```diff
@@ -3848,6 +3845,46 @@ static gps_mask_t processPQTMIMU(int count, char *field[],
...
+    mask |= NAVDATA_SET;
+
+    session->newdata.mode = MODE_3D;
+    mask |= MODE_SET;
+
+    GPSD_LOG( LOG_DATA, &session->context->errout,
+        "PQTMGPS: time=%d, lat=%.9f lon=%.9f",
+        atoi(field[1]),
+        session->newdata.latitude,
+        session->newdata.longitude
```
Comment (KaSroka) [inline drivers/driver_nmea0183.c]: I think you should log `wanglem` and `wangler` here.

**Review by KaSroka — APPROVED** [review]
Looks good, just one small comment.

File: drivers/driver_nmea0183.c (hunk 1 above)
Comment (Qbicz) [reply drivers/driver_nmea0183.c]: Fixed

**Pull request merged**